src/shared/locales/all.js

```diff
@@ -63,6 +65,12 @@ const resources = {
       ...appLocalePtBR
     }
   },
+  'ru-RU': {
```

Comment:
I think this locale key is wrong, you should use `ru` instead of `ru-RU`.

https://electronjs.org/docs/api/locales

> ru | Russian